pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -143,6 +143,12 @@ func (gc *GarbageCollector) attemptToDeleteWorker() bool {
 	}
 	err := gc.attemptToDeleteItem(n)
 	if err != nil {
+		// TODO: remove this block when gc starts using dynamic RESTMapper.
+		if restMappingError, ok := err.(*restMappingError); ok {
+			glog.Warning(restMappingError)
```

Comment:
What about `utilruntime.HandleError`?  You could (probably do?) wire that up for outside monitoring.